test/custom.test.js

```diff
@@ -40,3 +40,7 @@ it('works alongside global usage query', () => {
     var list = browserslist('> 10% in my stats, > 1%', { stats: usage });
     expect(list.length > 1).toBeTruthy();
 });
+
+it('take stats from browserslist-stats.json', () => {
+    expect(browserslist('> 5% in my stats')).toEqual(['ie 8']);
```

Comment:
Why this test are working? You didn’t pass a `file` option, so Browserslist should not know where to find.